src/main/java/core/basesyntax/MainApp.java

```diff
@@ -12,5 +12,16 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine[] machines = new Machine[]{truck, bulldozer, excavator};
+        for (Machine m : machines) {
```

Comment:
let's create more understandable name of the variable instead of `m`

Reply:
Good idea, I did